vcbuild.bat

```diff
@@ -208,6 +212,16 @@ if "%test%"=="test-simple" set test_args=%test_args% simple
 if "%test%"=="test-message" set test_args=%test_args% message
 if "%test%"=="test-gc" set test_args=%test_args% gc
 if "%test%"=="test-all" set test_args=%test_args%
+if "%test%"=="test-v8" goto test-v8
+if "%test%"=="test-v8-intl" goto test-v8-intl
+
```

Comment:
Note that the control here was falling through to :build-node-weak before your change. Now, it will fall through to :test-v8, causing v8 tests to **always** run.

You can either add a goto :build-node-weak after these checks or move :test-v8 and :-test-v8-intl somewhere below where they are not in the default control flow path, essentially making them subroutines.